cluster/cluster.yaml

```diff
@@ -1569,6 +1569,24 @@ Resources:
             - Action: ["s3:PutObject"]
               Effect: "Allow"
               Resource: 'arn:aws:s3:::{{.Cluster.ConfigItems.logging_s3_bucket}}/*'
+{{- if ne .Cluster.ConfigItems.telemetry_infrastructure_logs_bucket "" }}
+        - PolicyName: AllowCentralTelemetryInfrastructureS3BucketAccess
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+            - Action: ["s3:ListBucket"]
+              Effect: "Allow"
+              Resource: 'arn:aws:s3:::{{.Cluster.ConfigItems.logging_infrastructure_s3_bucket}}'
+            - Action: ["s3:GetObject"]
+              Effect: "Allow"
+              Resource: 'arn:aws:s3:::{{.Cluster.ConfigItems.logging_infrastructure_s3_bucket}}/*'
+            - Action: ["s3:PutObject"]
+              Effect: "Allow"
+              Resource: 'arn:aws:s3:::{{.Cluster.ConfigItems.logging_infrastructure_s3_bucket}}/*'
+            - Action: [ "s3:PutObjectAcl" ]
```

Comment:
Can we merge these into the same statement?

Reply:
merged